recipes/qt/all/conanfile.py

```diff
@@ -0,0 +1,665 @@
+            self.options.with_openal = False
+
+        if self.settings.os != "Linux":
+            self.options.with_libalsa = False
+
+        if self.options.qtwebengine:
+            if not self.options.shared:
+                raise ConanInvalidConfiguration("Static builds of Qt Webengine are not supported")
+
+            if tools.cross_building(self.settings, skip_x64_x86=True):
+                raise ConanInvalidConfiguration("Cross compiling Qt WebEngine is not supported")
+
+            if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
+                raise ConanInvalidConfiguration("Compiling Qt WebEngine with gcc < 5 is not supported")
+
+        if self.settings.os == "Android" and self.options.opengl == "desktop":
+            raise ConanInvalidConfiguration("OpenGL desktop is not supported on Android. Consider using OpenGL es2")
+
+        if self.settings.os != "Windows" and self.options.opengl == "dynamic":
+            raise ConanInvalidConfiguration("Dynamic OpenGL is supported only on Windows.")
+
+        if self.options.with_fontconfig and not self.options.with_freetype:
+            raise ConanInvalidConfiguration("with_fontconfig cannot be enabled if with_freetype is disabled.")
+
+        if self.settings.os == "Macos":
+            del self.settings.os.version
+
+        if self.options.multiconfiguration:
+            del self.settings.build_type
+
+        if not self.options.with_doubleconversion and str(self.settings.compiler.libcxx) != "libc++":
+            raise ConanInvalidConfiguration('Qt without libc++ needs qt:with_doubleconversion. '
+                                            'Either enable qt:with_doubleconversion or switch to libc++')
+
+        if tools.os_info.is_linux:
+            if self.options.qtwebengine:
+                self.options.with_fontconfig = True
+
+        def _enablemodule(mod):
+            if mod != 'qtbase':
+                setattr(self.options, mod, True)
+            for req in self._submodules[mod]["depends"]:
+                _enablemodule(req)
+
+        for module in self._submodules:
+            if module != 'qtbase' and getattr(self.options, module):
+                _enablemodule(module)
+
+    def requirements(self):
+        if self.options.openssl:
+            self.requires("openssl/1.1.1g")
+        if self.options.with_pcre2:
+            self.requires("pcre2/10.33")
+
+        if self.options.with_glib:
+            self.requires("glib/2.64.0@bincrafters/stable")
+        # if self.options.with_libiconv:
+        #     self.requires("libiconv/1.16")
+        if self.options.with_doubleconversion and not self.options.multiconfiguration:
+            self.requires("double-conversion/3.1.5")
+        if self.options.with_freetype and not self.options.multiconfiguration:
+            self.requires("freetype/2.10.1")
+        if self.options.with_fontconfig:
+            self.requires("fontconfig/2.13.91@conan/stable")
```

Comment:
Not ported yet

Reply:
https://github.com/conan-io/conan-center-index/pull/1756

Reply:
https://github.com/conan-io/conan-center-index/pull/1762 is now mergeed